Day001_Development_Setup/README.md

```diff
@@ -107,6 +107,15 @@ Running docker compose command "build"
 #24 DONE 0.0s
 ```
 
+- A successful```invoke build``` completion, will output a docker image for Nautobot, check this with ```docker ps```. If no images are visible, try the invoke process again and/or request help in the [Nautobot Slack](https://networktocode.slack.com/archives/C01UJ9ZQZ3D).
```

Comment:
Should be ```docker images``` for the check, right?

Reply:
I'm loathe to submit a pr for one word,, but I can hang onto it and add other updates if I find them

Reply:
Yeah good catch. The example uses `docker images` but my statement is incorrect /facepalm.

I've fixed in https://github.com/nautobot/100-days-of-nautobot/pull/6